### ISSUE TITLE
Add venv explanation?

### ISSUE THREAD

**Issue by Amustache** [clicepfl/roboclic-archived#23]
Not sure if you want to add it to the README or not, but it's often useful to teach people to "compartment" their environment.

**Comment by LucasTrg** [comment]
Agreed ! Might even add some stuff for the poetry enjoyers (although it's perhaps not really useful, as the repo is meant to be automated)
